### PR TITLE
add buckets 

### PR DESCRIPTION
**Pull request by Fab-T**
Adding module create buckets for input and output

Still need to complete before bucket creation with proper configuration like ACL.

### CODE REVIEW

File: deploy/aou/terraform/aou-in-bucket.tf [hunk 1]
```diff
@@ -0,0 +1,39 @@
+module "aou-in-bucket" {
```
Comment (rexwangcc) [inline deploy/aou/terraform/aou-in-bucket.tf:1]: Could we make it consistent with the other? i.e. `aou-in` <=> `aou-out` or `aou-input` <=> `aou-output`

File: deploy/aou/terraform/aou-in-bucket.tf [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  }
+
+  # Create one bucket and set ACLs
+  bucket_name = "aou-in-bucket"
+  versioning = true
+  bindings = {
+    storage_admins = {
+      role = "roles/storage.admin"
+      members = [
+        "group:hornet@broadinstitute.org",
```
Comment (rexwangcc) [inline deploy/aou/terraform/aou-in-bucket.tf:17]: hops might also need to be added either to admins or viewers?

File: deploy/aou/terraform/aou-in-bucket.tf [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+      ]
+    },
+    viewers_legacy = {
+      role = "roles/storage.legacyBucketReader"
+      members = [
+        "group:lantern@broadinstitute.org",
+      ]
+    }
+  }
+}
```
Comment (rexwangcc) [inline deploy/aou/terraform/aou-in-bucket.tf:39]: nit: new line at the end.

File: deploy/aou/terraform/aou-output-bucket.tf [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+      ]
+    },
+    viewers_legacy = {
+      role = "roles/storage.legacyBucketReader"
+      members = [
+        "group:lantern@broadinstitute.org",
+      ]
+    }
+  }
+}
```
Comment (rexwangcc) [inline deploy/aou/terraform/aou-output-bucket.tf:39]: nit: new line at the end.

File: deploy/aou/terraform/aou-in-bucket.tf (hunk 2 above)
Comment (Fab-T) [reply deploy/aou/terraform/aou-in-bucket.tf:17]: of course this is just example and I need more info, thus the document I shared

File: deploy/aou/terraform/aou-in-bucket.tf (hunk 1 above)
Comment (Fab-T) [reply deploy/aou/terraform/aou-in-bucket.tf:1]: yes I would like to get the proper naming convention thus the document I shared.

**Pull request merged**